Makefile.in

```diff
@@ -29,7 +29,7 @@ TEST_LIBS=$(LIBNAME1).a
 LDSHARED=$(CC)
 LDSHAREDFLAGS=-shared
 
-VER=2.1.0.beta1
+VER=2.1.1.beta2
```

Comment:
``2.1.1.beta2``
or
``2.1.0.beta2``
?
For beta versions, the last component of the version must also be updated?

Reply:
I will update the minor version for each release to avoid potential confusion, and alpha/beta/rc are all releases and not snapshots.

I follow a similar release model as for example MariaDB.

This also means that 2.1.7 might be a stable release, followed by a beta 2.1.8, before a stable 2.1.9 after that again.
Without increasing the version number, we could potentially get bug reports that look like a stable release when it was in fact a beta.

I know this is not the most common way to do versioning, but IMO it has merit.